model/textparse/openmetricsparse.go

```diff
@@ -94,14 +94,49 @@ type OpenMetricsParser struct {
 	exemplarVal   float64
 	exemplarTs    int64
 	hasExemplarTs bool
+
+	// lines ending with _created are skipped by default as they are
+	// parsed by the CreatedTimestamp method. The skipCT flag is set to
+	// false when the CreatedTimestamp method is called.
```

Comment:
This is no longer true. Also it's better to comment in public place and do it once. Let's kill this?
```suggestion
```